lib/utils.js

```diff
@@ -7,18 +7,18 @@ var changeTypeLookup = {
   abbreviation: abbreviation
 };
 
-var isNullOrUndefined = function(val) {
+exports.isNullOrUndefined = function(val) {
```

Comment:
Don't we need to use `module.exports` here? See https://blog.tableflip.io/the-difference-between-module-exports-and-exports/